### DISCUSSION TITLE
Install extension via vsix file - does not work

### DISCUSSION THREAD

**Discussion by whati001** [coder/code-server#4514] [Q&A]
Hi,

it seems like I am not able to install extensions via vsix files.
The process traces that the ext was installed, but it does not appear in the side bar.
At least just for an half of an second and than it is gone.

I am using the docker images and the ext is downloaded from the MSStore.

thx for helping me

**Comment by jsjoeio** [comment]
What happens if you reload? Does the ext load then?

**Comment by abhay-ranawat** [comment]
Maybe it's happening due to root(`root/.local/share/vscode`) vs user directory (`$HOME/.local/share/vscode`), where you install extensions in Dockerfile using root, but then change user to start code-server

**Reply by jsjoeio** [reply]
^Good idea! Maybe this is it
